src/FieldWarning/Assets/Model/Game/PlayerData.cs

```diff
@@ -35,10 +36,18 @@ public class PlayerData
                     _session = value;
             }
         }
+        private DeckConfig _deckConfig;
 
-        public Deck Deck { get; private set; } = new Deck();
+        public Deck Deck { get; private set; }
 
         public float Money = 1000;
         public float IncomeTick = 7;
+        
+        public PlayerData(string name)
+        {
+            Name = name;
+            _deckConfig = ConfigReader.FindDeckConfig(Name);
```

Comment:
As mentioned above, I would actually call this in MatchSession and then pass the parsed deck from MatchSession to here. The reason is, in the final configuration we'd load the decks in a menu scene, then pass them to the MatchSession when we load a match scene.